docs/en/sql-reference/data-types/map.md

```diff
@@ -75,6 +76,54 @@ SELECT CAST(([1, 2, 3], ['Ready', 'Steady', 'Go']), 'Map(UInt8, String)') AS map
 └───────────────────────────────┘
 ```
 
+## Subcolumn Map.keys {#subcolumn-keys}
+
+To read all keys of a `Map` you can use the subcolumn `keys`, which doesn't read the whole column.
```

Comment:
```suggestion
To optimize the `Map` column processing, in some cases you can use the `keys` and 'values' subcolumns instead of reading the whole column.
```